pennylane/operation.py

```diff
@@ -114,6 +114,7 @@
     Non-Gaussian CV operations and observables are currently only supported via
     the finite-difference method of gradient computation.
 """
+# pylint:disable=access-member-before-definition
```

Comment:
Sorry, I am cheating here - codecov mixes lines up so I don't know where the issue is